syntax = "proto3";

package organizations.mesh;

message SetNewCustomerConfigurationResponse {
  string vehicle_id = 1;
  int32 set_customer_configuration_result = 2;
  string aux_data = 3;
}